airflow/settings.py

```diff
@@ -722,6 +722,22 @@ def import_local_settings():
         log.info("Loaded airflow_local_settings from %s .", airflow_local_settings.__file__)
 
 
+def mask_conf_values():
+    from airflow.utils.log.secrets_masker import mask_secret
+
+    for section, key in conf.sensitive_config_values:
+        try:
+            value = conf.get(section, key)
+        except AirflowConfigException:
+            log.warning(
+                "Could not retrieve value from section %s, for key %s. Skipping redaction of this conf.",
+                section,
+                key,
+            )
+            continue
+        mask_secret(value)
```

Comment:
This feels like it should be method on Configuration, not a few method in settings module.

Could you move it and the tests please?

(Calling conf.mask_secrets from withing settings is still okay I think)

Reply:
I cannot call the mask_secret from configuration. It gets a circular import there

Reply:
Okay let me do that

Reply:
@ashb the challenge in doing that would be importing the `mask_secret` inside configuration.py.
It would lead to a circular import.

Reply:
Ah right k. Well we could choose to import `mask_secret` only inside this function. I guess it's fine here where it is

Reply:
Wait, the import is already inside this function anyway, so it wouldn't create a circular import (only top level imports can create circular imports. (Circular dependency, sure, but that's a fact of life in python really.)

Reply:
Alright, slightly confused with your comment. I have imported mask_secret only in here. Should I make some changes or this is alright?

Reply:
I'm saying, that since the import is here, that we can move this function to be a method on our Configuration class in airflow/configuration.py without causing any import cycles. Importing things inside a function to avoid import loops is a pattern we do a lot in Airflow.

Reply:
Let me try that out, makes sense.